bin/load_db_scxa_dimred.sh

```diff
@@ -5,15 +5,16 @@
 # parameterisations, and loads it into the scxa_coords table of AtlasProd.
 set -e
 
-scriptDir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+scriptDir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}" )" &> /dev/null && pwd )
 source $scriptDir/db_scxa_common.sh
 
 dbConnection=${dbConnection:-$1}
 EXP_ID=${EXP_ID:-$2}
 DIMRED_TYPE=${DIMRED_TYPE:-$3}
 DIMRED_FILE_PATH=${DIMRED_FILE_PATH:-$4}
 DIMRED_PARAM_JSON=${DIMRED_PARAM_JSON:-$5}
-SCRATCH_DIR=${SCRATCH_DIR:-"$(dirname ${DIMRED_FILE_PATH})"}
+#SCRATCH_DIR=${SCRATCH_DIR:-"$(dirname ${DIMRED_FILE_PATH})"}
+SCRATCH_DIR=${SCRATCH_DIR:-"$DIMRED_FILE_PATH"}
```

Comment:
Have you tested this after changing dirname to full path ?

Reply:
How is this any different from the old code? The only thing I can see is that it will be more prone to creating files in unexpected locations.

Reply:
Hi @anilthanki Yes, it is tested and working